### PR TITLE
Register smutex.is-a.dev

### PR DESCRIPTION
**Pull request by ghost**
Register smutex.is-a.dev with CNAME record pointing to smalorek.github.io.

### CODE REVIEW

**Review by triach-rold — CHANGES_REQUESTED** [review]
Unreachable.

**Comment by Smalorek** [comment]
?

File: domains/smutex.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Smalorek",
+           "email": "Smalorek@proton.me",
+           "discord": "286178186645995522"
+        },
+    
+        "record": {
+            "CNAME": "smalorek.github.io"
```
Comment (Stef-00012) [inline domains/smutex.json:9]: You should complete your website first

i visited your repo and saw only a file with
```html
<h1>Test</h1>
```

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Comment by Smalorek** [comment]
ok 👍

**Comment by orxngc** [comment]
Closing as stale

**Comment by orxngc** [comment]
Closing as stale

**Pull request closed without merging**